src/test/java/stringcalculator/StringCalculator.java

```diff
@@ -1,40 +1,15 @@
 package stringcalculator;
 
-import java.util.Arrays;
-import java.util.regex.Matcher;
-import java.util.regex.Pattern;
-
 public class StringCalculator {
 
-    private String text;
-
-    private static StringValidation stringValidation = new StringValidation();
-    private static DelimiterParser delimiterParser = new DelimiterParser();
+    private final StringValidation stringValidation;
 
     public StringCalculator(){
+        this.stringValidation = new StringValidation();
     }
```

Comment:
```suggestion
    public StringCalculator(String text){
        this.stringValidation = new StringValidation(text);
        this.text = new Parser(text).getText;
        ...
    }
```

단편적인 예를 표현한 것이지만 상태 값을 활용해 보면 어떨까요?